bufferserver/src/main/java/com/datatorrent/bufferserver/server/Server.java

```diff
@@ -165,13 +178,12 @@ public static void main(String[] args) throws Exception
   @Override
   public String toString()
   {
-    return identity;
+    return getClass().getSimpleName() + '@' + Integer.toHexString(hashCode()) + "{address=" + address + "}";
```

Comment:
No, it is mostly for consistency with the Object toString(). Most likely address (port) will not be reused when the Server is started again on the same node.